packages/common/src/eips.ts

```diff
@@ -542,4 +542,13 @@ export const EIPs: EIPsDict = {
       },
     },
   },
+  7685: {
+    comment: 'General purpose execution layer requests',
+    url: 'https://eips.ethereum.org/EIPS/eip-7685',
+    status: Status.Draft,
+    // TODO: Set correct minimum hardfork
+    minimumHardfork: Hardfork.Shanghai,
```

Comment:
Guess this also can be kept as "Shanghai".

Reply:
I'm updating it to Cancun for now until we decide how to handle the potentially undefined fields for `blobGasUsed`, `excessBlobGas`, and `parentBeaconBlockHash`, that exist in the blockheader values array if we don't activate 4844 and 4788 alongside 7685.